src/block-management/block-manager.js

```diff
@@ -23,6 +22,8 @@ import EventEmitter from 'events';
 const keyboardDidShow = 'keyboardDidShow';
 const keyboardDidHide = 'keyboardDidHide';
 
+const { height: fullHeight } = Dimensions.get( 'window' );
```

Comment:
From [Dimensions.get](https://facebook.github.io/react-native/docs/dimensions.html#get):

> Although dimensions are available immediately, they may change (e.g due to device rotation) so any rendering logic or styles that depend on these constants should try to call this function on every render, rather than caching the value

Reply:
Good point @koke.
Fixed.